Library/Homebrew/cmd/audit.rb

```diff
@@ -23,65 +26,51 @@
 require "date"
 
 module Homebrew
+  RUBY_2_OR_LATER = RUBY_VERSION.split(".").first.to_i >= 2
```

Comment:
Yeah. Same with the test for whether encodings are supported. I'd rather defer that to another PR, though.